context.go

```diff
@@ -164,7 +164,7 @@ func (ctx *Context) CORS(domains ...string) *Context {
 	origins := make([]string, 0)
 	if len(domains) > 0 {
 		origins = domains
-	} else if headerOrigin := ctx.Request.Header.Peek(hOrigin); headerOrigin != nil && len(headerOrigin) > 0 {
+	} else if headerOrigin := ctx.Request.Header.Peek(hOrigin); headerOrigin != nil {
```

Comment:
headerOrigin may be allocated with len(0), so if you trying to minimize CMPQ/JNE's I'd like to leave the len > 0 part instead of != nil

Reply:
```
context.go:167:62:warning: should omit nil check; len() for nil slices is defined as zero (S1009) (megacheck)
```

Reply:
Yes, so just replace != nil with len > 0